cmd/kyma/init/function/function.go

```diff
@@ -38,7 +39,7 @@ Use the flags to specify the initial configuration for your Function or to choos
 		},
 	}
 
-	cmd.Flags().StringVar(&o.Name, "name", defaultName, `Function name.`)
+	cmd.Flags().StringVar(&o.Name, "name", generateRandomFunctionName(), `Function name.`)
```

Comment:
It would be better to only generate a name if the user is not providing the flag because random generation is costly.

Reply:
This PR is still `Work In Progress` ;) We know that, but thanks for your suggestions

Reply:
I see, in that case please create `Draft` pull requests so that reviewers are not being notified.